src/commands/suggestions/suggest.ts

```diff
@@ -0,0 +1,90 @@
+import { ApplyOptions } from '@sapphire/decorators';
+import { Message, MessageEmbed } from 'discord.js';
+import { Args, container} from '@sapphire/framework';
+import { SubCommandPluginCommand, SubCommandPluginCommandOptions } from '@sapphire/plugin-subcommands';
+import { addSuggestion, getSuggestions, suggestionStatesReadable, SuggestionState, updateSuggestionState, Suggestion, getAvailableStatesString } from '../../components/suggestions';
+import { EMBED_COLOUR } from '../../utils/embeds';
+import _ from 'lodash';
+
+const RESULTS_PER_PAGE = 15;
+
+@ApplyOptions<SubCommandPluginCommandOptions>({
+    aliases: ['suggestions'],
+    description: 'Handles suggestion functions',
+    detailedDescription: `This command will forward a suggestion to the CSC Discord Mods. \n
+    Please note that your suggestion is not anonymous, your Discord username and ID will be recorded. \n
```

Comment:
For these two lines, I think revert `\n` back to `\` so they will be on the same line?

Reply:
oops for some reason my changes disappeared